geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnectionFactory.java

```diff
@@ -41,8 +41,8 @@ private synchronized void initializeAuthenticatorsMap() {
       return;
     }
     authenticators = new HashMap<>();
-    ServiceLoader<StreamAuthenticator> loader = ServiceLoader.load(StreamAuthenticator.class);
-    for (StreamAuthenticator streamAuthenticator : loader) {
+    ServiceLoader<Authenticator> loader = ServiceLoader.load(Authenticator.class);
+    for (Authenticator streamAuthenticator : loader) {
```

Comment:
may we want to remove stream from variable name

Reply:
The "stream" concept is fine in the cache.tier.sockets package - there are other uses of it there.